core/templates/pages/skill-editor-page/editor-tab/skill-prerequisite-skills-editor/skill-prerequisite-skills-editor.directive.ts

```diff
@@ -18,7 +18,7 @@
 
 require('components/skill-selector/select-skill-modal.controller.ts');
 require(
-  'components/skill-selector/skill-selector.directive.ts');
```

Comment:
component import not required.

Reply:
Interesting, why is that?

Reply:
This is because of how we register components in angularjs vs angular. In the angular the component is registered in a separate file called module.ts however in AJS, the component is registered in the file itself. Hence until unless the file is required, the line `angular.module('oppia').directive(...)` is not run. That said, I think its fine to keep it right now. Doesn't cause any problems.

Reply:
Ah I see now. Leaving it for now since this components is in AJS too, so that might be tricky.